docs/source/onboarding_resources/tutorial/index.ipynb

```diff
@@ -0,0 +1,1653 @@
+{
```

Comment:
<p>"For&nbsp;diarrheal diseases," is not a sentence</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1579/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>an astute observation -- oops</p>